src/ru/skypro/Main.java

```diff
@@ -1,8 +1,84 @@
 package ru.skypro;
 
+import java.util.Arrays;
+
 public class Main {
+    private static final Employee[] employees = new Employee[10];
 
+    public static void FullName(Employee[] e) {
+        String fullname;
+        for (Employee fu : employees) {
+            fullname = fu.fullName;
+            System.out.println("Ф.И.О. - " + fullname);
+        }
+    }
+    public static void Sum (Employee[] e) {
+        double sum;
+        double summ = 0;
+        for (Employee su : employees) {
+            sum = su.salary;
+            summ += sum;
+        }
+        System.out.println("Сумма зарплат - " + summ);
+    }
+    public static void maxmin (Employee[] e) {
+        double min = 100000;
+        double max = 0;
+        for (Employee element : employees) {
+            if (element.salary > max) {
+                max = element.salary;
+            }
+            if (element.salary < min) {
+                min = element.salary;
+            }
+        }
+        System.out.println("Минимальная зарплата - " + min);
+        System.out.println("Максимальная зарплата - " + max);
+    }
+    public static void srednee (Employee [] e) {
```

Comment:
лучше именовать с использованием английским слов, всегда можно проверить себя в словаре
```suggestion
    public static void printAverageSalary(Employee[] employees) {
```